web/sendgrid_webhooks.py

```
@@ -0,0 +1,166 @@
+                "email_spam_report_count",
+                "last_email_event",
+                "last_email_event_time",
+                "email_last_event_data",
+            ]
+        )
+
+    # Send notification to Slack
+    send_sendgrid_event_to_slack(event, email, event_type, user)
+
+
+def send_sendgrid_event_to_slack(event, email, event_type, user=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `send_sendgrid_event_to_slack`

Add return type annotation: `None`

(ANN201)